easybuild/easyconfigs/j/julia/julia-1.0.eb

```diff
@@ -0,0 +1,20 @@
+# Recipe for linux, x86_64 created by Samuel Omlin (CSCS), September 12, 2018
+easyblock = 'PackedBinary'
+
+name = 'julia'
+version = '1.0.0'
```

Comment:
the file name does not match the version number

Reply:
good catch!